endoreg_db/views/requirement/evaluate.py

```
@@ -0,0 +1,279 @@
+from endoreg_db.models.requirement.requirement_set import RequirementSet
+from endoreg_db.models.medical.patient.patient_examination import PatientExamination
+
+
+from rest_framework import status
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+import logging
+
+logger = logging.getLogger(__name__)
+@api_view(['POST'])
+def evaluate_requirements(request):
```

Pylint (3.3.7)

[refactor] Too many local variables (22/15)

(R0914)